<!DOCTYPE HTML>
<!--
	Helios by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<!-- Google tag (gtag.js) -->
		<script async src="https://www.googletagmanager.com/gtag/js?id=G-GEHVE1VMSG"></script>
		<script>
  		window.dataLayer = window.dataLayer || [];
  		function gtag(){dataLayer.push(arguments);}
  		gtag('js', new Date());

  		gtag('config', 'G-GEHVE1VMSG');
		</script>
		<title>Ecological Data Science - Projects</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="no-sidebar is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<div id="header">

					<!-- Inner -->
						<div class="inner">
							<header>
					   					<h1>The Leverhulme Programme for Doctoral<br>Training in Ecological Data Science</h1>
								<hr />
					   					<p>University of Glasgow</p>
							</header>
						</div>

					<!-- Nav -->
						<nav id="nav">
							<ul>
								<li><a href="index.html">Home</a></li>
								<li><a href="about.html">About</a></li>
								<li><a href="projects.html">Projects</a></li>
								<li><a href="faqs.html">FAQs</a></li>
								<li><a href="apply.html">Apply</a></li>
								<li><a href="contact.html">Contact</a></li>
							</ul>
						</nav>
				

				</div>

			<!-- Main -->
				<div class="wrapper style1">

					<div class="container">
						<article id="main" class="special">
							<h2>Project descriptions</h2>
							<p>The listed projects represent the available projects for the 2024 cohort. In year 1, students will undertake a group rotation project in 3 of the listed projects, before selecting a single project as their PhD topic.</p>
						</article>
						<br>

						<div class="row">
							<article class="col-4 col-12-mobile special">
								<div class="image featured"><img src="images/faust.jpg" alt="a mouse on a branch with berries"/></div>

								<h3>Inferring ecological interactions from metagenomics</h3>
								<p>
								<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/bohvm/staff/christinafaust">Dr. Christina Faust</a></h4>
								</p>
  							<details>
    							<summary>Project summary</summary>
								<p>Advances in genomic sequencing have enabled the characterisation of entire viral communities within individual hosts without the need for isolation or a priori information on species. Metagenomic surveys have accelerated the discovery of new pathogen species across all kingdoms of life, vastly expanding our knowledge of global pathogen diversity. However, metagenomic approaches have generally been underutilised for advancing our understanding of ecological interactions. This project will leverage machine learning methods to use viral host predictor models to classify plant vs rodent viruses and to identify vector-borne viruses; this will allow them to characterise the community dynamics of viruses among rodents, identify demographic and ecological drivers of virome composition, and construct ecological interaction networks.
								</p>
								</details>
							</article>

							<article class="col-4 col-12-mobile special">
								<div class="image featured"><img src="images/torney.jpg" alt="a blue background with lines and dots" /></div>

								<h3> Modelling mobile animal groups as attention-based graph neural networks
								</h3>
								<p>
								<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/mathematicsstatistics/staff/colintorney">Prof. Colin Torney</a></h4>
								</p>
  							<details>
    							<summary>Project summary</summary>
								<p>Collective movement is ubiquitous throughout the natural world; from fish schools, marching ants, to herds of wildebeest, living organisms live and move collectively. Understanding how animal groups are able to achieve large scale coordinated motion in the absence of centralised control is an important challenge for movement ecology. Recent developments in artificial intelligence have made it possible to develop and train intelligent algorithms that are able to process large complex datasets and extract  meaningful insight from these data. A new class of machine learning models, graph neural networks, present an exciting opportunity for understanding interacting systems including animal groups. These models can be trained on simulated or empirical data and can learn to predict the behaviour of complex interacting systems. This project seeks to develop a framework for using attention-based graph neural networks to model collective movement of animal groups using data from UAV studies of wildebeest, caribou, and bison. A core aim of the project will be to investigate the connections between information processing in animal groups and the emergent intelligence of attention-based neural networks.
								</p>
								</details>
							</article>

							<article class="col-4 col-12-mobile special">
								<div class="image featured"><img src="images/haydon.jpg" alt="visualisation of coronavirus" /></div>

								<h3> Estimating the size of partially observed disease outbreaks from sequence data
								</h3>
								<p>
								<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/bohvm/staff/danielhaydon">Prof. Daniel Haydon</a></h4>
								</p>
  							<details>
    							<summary>Project summary</summary>
								<p>Many important pathogens have now been shown to evolve fast enough that pathogen genomic diversity accumulates over transmission events during even a relatively short outbreak.  The increasing affordability of pathogen genome sequence data and the speed at which it can accumulate provides many potential opportunities to study the epidemiology of disease outbreaks in news way.  We are particularly interested in how this diversity can be used to estimate the overall number of cases generated over an outbreak.  It is inevitable that sequence data will be acquired from only a subset of infected cases, so it is necessary to account quantitatively for cases for which genomic samples were not collected.  Different approaches have been described in the literature, but they often require or assume detailed knowledge of the underlying epidemiology and host-pathogen population structure and dynamics.  Here we propose to develop a semi-parametric and non-mechanistic approach for estimating the size of a partially observed (i.e. incompletely genomically sampled) outbreak from monophyletic pathogen sequence data that uses the internal and external lengths of the branches of a phylogenetic tree constructed from the genome data. We will need to develop a simple and unbiased estimator of outbreak size applicable to inevitably generally quite low observation frequencies, and that can accommodate various forms of common biases and heterogeneities in the observation process.  
								</p>
								</details>
							</article>



						</div>

						<hr />
						
						<div class="row">

							<article class="col-4 col-12-mobile special">
								<div class="image featured"><img src="images/dirk.jpg" alt="Herd of sheeps" /></div>


								<h3>Mathematical and statistical modelling of coordinated animal behaviour</h3>
								<p>
								<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/mathematicsstatistics/staff/dirkhusmeier">Prof. Dirk Husmeier</a></h4>
								</p>
  							<details>
    							<summary>Project summary</summary>
								<p>
								Understanding collective movement is of key interest in ecology, as animal migration connects ecological communities and mediates their diversity and stability. The purpose of this project is to develop advanced statistical and computational tools to learn about the key driving factors of collective movement. The student will study a flock of 60 sheep, and formulate hypotheses about their collective movement with a set of mathematical models. The animals are equipped with GPS and accelerometers, from which high-resolution time series of their positions are available. This unique dataset will allow the student to understand the emergence of coordinated large- scale movement from individual decisions.
								The PhD project aims to contribute to the study of collective animal behaviour and movement, which sees the emergence of coordinated large scale phenomena from individual-based processes, such as switches between internal behavioural states and local interactions with other group members and the environment. The student will develop a novel suite of detailed mathematical models to describe these processes, and address the important challenges of sensitivity analysis and statistical inference: which model parameters have a significant effect on the large-scale phenomena of interest, how can they be estimated, and how can their uncertainty be quantified? The student's work will particularly focus on the study of a flock of free-roaming sheep in Patagonia, with location data via GPS tracking combined with accelerometer and magnetometer recordings and a detailed environmental map of vegetation and topography. This not only provides an excellent testbed for the evaluation of our new models and inference methods, but is also a unique resource for the elucidation of collective animal movement processes per se.
								</p>
								</details>
							</article>

							<article class="col-4 col-12-mobile special">
								<div class="image featured"><img src="images/hopcraft.jpg" alt="wildebeest with collar" /></div>
								<h3> Understanding the impact of human presence on migratory wildebeest using edge machine learning</h3>
								<p>
								<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/bohvm/staff/granthopcraft">Prof. Grant Hopcraft</a></h4>
								</p>
  							<details>
    							<summary>Project summary</summary>
								<p>Understanding how animals respond to human presence is a vital component of balancing conservation and development.  This project will use a combination of GPS tracking data, accelerometers, edge machine learning, and AI remote sensing to understand how human presence and tourism infrastructure affects the movement and behaviour of wildebeest in the Serengeti National Park, Tanzania.  The project will use a combination of statistical and mechanistic modelling approaches to understand how wildebeest alter their movement patterns and activity budgets in response to tourism and will specifically seek to develop strategies for high-volume, low impact, sustainable tourism practices.
								</p>
								</details>
							</article>


							<article class="col-4 col-12-mobile special">
								<div class="image featured"><img src="images/illian.jpg" alt="green trees under white clouds" /></div>
								<h3>Combining data sources in multi-layer point process models to distinguish habitat preferences and dispersal limitation in tropical forest plant communities</h3>
								<p>
								<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/mathematicsstatistics/staff/janineillian">Prof. Janine Illian</a></h4>
								</p>
  							<details>
    							<summary>Project summary</summary>
								<p>There is an interest in understanding the specific habitat preferences of different rainforest species and which ecological processes dominate in shaping plant communities.   Large data sets exist from plots across multiple tropical forest sites that detail the exact spatial location of each tree as well as environmental covariates such as topography and soil nutrients. Some plots have a nested grid of seed traps that accumulate data on the spatial pattern of seed arrival through time.   Analyses of these point patterns have shown that rainforest trees cluster in space. The processes that determine the emergence of these spatial structures are complex, but potentially include spatial aggregation on more favourable habitats, competition between species, non-random seed dispersal linked to the distribution of adult trees and spatially structured patterns of individual survival. Since these processes operate contemporaneously, they cannot be distinguished based on a single point pattern.   For this project, the additional data have been collected through seed traps and seedling maps which can be used to gain a better understanding of seed dispersal.   The project will develop multi-layered, multi-likelihood spatial point process models that combine the different data sources using hierarchical Bayesian methods fitted through computationally efficient model fitting approaches.      
								</p>
								</details>
							</article>





						</div>

						<hr />
						<div class="row">
						
							<article class="col-4 col-12-mobile special">
								<div class="image featured"><img src="images/jason.png" alt="a surface plot and a 3d scatter plot" /></div>
								<h3> Useful ecological predictions in a changing and data-deficient world</h3>
								<p>
								<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/bohvm/staff/jasonmatthiopoulos">Prof. Jason Matthiopoulos</a></h4>
								</p>
  							<details>
    							<summary>Project summary</summary>
								<p>The predictions of applied tools such as species distribution modelling and population viability analysis are increasingly used to answer what-if questions about anthropogenic risk. This confronts us with two interlocking challenges: transferability and data-sparsity. How can we use data from a given place, time and species, to make useful (i.e., both accurate and precise) predictions about related species at some other place, in the future? This project will exploit the convergence between mechanistic ecological theory and modern statistical inference to drastically increase the predictive reach of ecological models in a world where change is the rule, not the exception. It will achieve this by combining models and data efficiently, according to six principles: 1) Space-for-time substitution, 2) Using contrasting environmental scenarios,  3) Using data from contrasting population scenarios, 4) Connecting habitat to population change, 5) Fully propagating uncertainty to predictions, 6) Sharing insights and inference between species.  
								</p>
								</details>
							</article>



						<article class="col-4 col-12-mobile special">
							<div class="image featured"><img src="images/morales.png" alt="a strange attractor" /></div>
							<h3> Linking behaviour and space use with population dynamics</h3>
							<p>
							<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/bohvm/staff/juanmorales">Prof. Juan Morales</a></h4>
							</p>
  							<details>
    							<summary>Project summary</summary>
							<p>Ecology is undergoing a revolutionary transformation with a flood of new types of data from GPS tracking tags and biologging sensors that are providing an unprecedented level of detail about what animals are doing and where they are doing it. This wealth of data has been used to create step-changes in our understanding of the scales at which animals move, what habitats are selected, and the choices that determine behaviours such as vigilance and foraging. However, this new data could also be used to address new and larger-scale questions about population dynamics. In principle, forging quantitative links between fine-scale decision-making and individual fitness could permit projections up to a population scale, enabling predictions about demography and even the dynamics of ecological communities. The challenge is that the analytical methodology required to make this link is in a relatively undeveloped state, and these links are rarely rigorously established. Furthermore, the enormous volume of data being collected requires innovative and efficient ways to perform inference and uncertainty quantification. Our goal is to develop a comprehensive framework to link behaviour, space use, and population dynamics exploiting the recent technological advances in fine-scale data acquisition with cutting-edge statistical methodology. Application of this framework would provide us with much deeper and wide-ranging insights into the drivers of demographic change and will allow the development of pre-emptive conservation strategies for endangered populations and management plans for free ranging livestock.
							</p>
								</details>
						</article>



							<article class="col-4 col-12-mobile special">
								<div class="image featured"><img src="images/clippele.jpg" alt="underwater photo" /></div>
  
								<h3> Interactive machine learning for the automated detection of marine biological sounds to understand how biodiversity changes in time and space</h3>
								<p>
								<h4>Primary supervisor: <a href="https://www.gla.ac.uk/schools/bohvm/staff/laurencedeclippele">Dr. Laurence De Clippele</a></h4>
								</p>
  							<details>
    							<summary>Project summary</summary>
								<p>This project aims to automate marine biodiversity monitoring, using a combination of long-term deep-sea image and acoustic datasets, by developing a user-friendly interactive machine learning (IML) software tool.   Deep-sea habitats, such as cold-water coral reefs and sponge grounds, are hotspots of biodiversity and are fundamental to our planetâ€™s health, resilience and food provision, all of which are closely linked to human well-being. Increasing amounts of long-term image and sound (acoustic) datasets are being collected to understand how their biodiversity is affected by natural-, human- and climate-driven changes. Analysing these datasets manually for the presence of species-specific sounds, produced by e.g. fish and crustaceans, can be prohibitively time-consuming. While IML has proven useful for species detection in underwater images, using IML to detect fish and crustacean sounds is a new and exciting field of research. The PhD student will be part of international projects focusing on understanding the importance of deep-sea habitats and their restoration for biodiversity and commercially important species. For end-users, such as biologists and ecologists, to trust and use the IML tool, the PhD student will develop skills in understanding the ecological relevance of the outputs and science communication.  
								</p>
								</details>
							</article>



						



					</div>
						<hr />

				</div>

				<div id="footer">
					<div class="copyright">
						<small>&copy; 2024 University of Glasgow.</span> <span>Designed by <a href="http://freehtml5.co/" target="_blank">FreeHTML5.co</a>. </span> <span> Banner image by <a href="https://unsplash.com/@danotis?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Dan Otis</a> on <a href="https://unsplash.com/photos/aerial-view-of-pine-trees-in-mist-OYFHT4X5isg?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>.</span> </small>
						<small>Photo credits:  <a href="https://unsplash.com/photos/a-mouse-on-a-branch-with-berries-TOHHuI7cot8?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">A mouse on a branch with berries</a> by <a href="https://unsplash.com/@hoops1972?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Glen Hooper</a>.
						<a href="https://unsplash.com/photos/a-blue-background-with-lines-and-dots-xuTJZ7uD7PI?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">A blue background with lines and dots</a> by <a href="https://unsplash.com/@choys_?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Conny Schneider</a>.
						<a href="https://unsplash.com/photos/visualization-of-the-coronavirus-causing-covid-19-rnr8D3FNUNY?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Visualization of the coronavirus causing COVID-19</a> by <a href="https://unsplash.com/@fusion_medical_animation?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Fusion Medical Animation</a>.
						<a href="https://unsplash.com/photos/herd-of-sheeps-soslo4sEmBA?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Herd of sheeps</a> by <a href="https://unsplash.com/@charlottevenema?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Charlotte Venema</a>.
						Wildebeest with collar by <a href="https://danielrosengren.se/">Daniel Rosengren</a>.
						<a href="https://unsplash.com/photos/green-trees-under-white-clouds-GMjKxyV9oSo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Green trees under white clouds</a> by <a href="https://unsplash.com/@boudewijn_huysmans?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Boudewijn Huysmans</a>.

							</small>
	 							<!-- Photo by <a href="https://unsplash.com/@choys_?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Conny Schneider</a> on <a href="https://unsplash.com/photos/a-blue-background-with-lines-and-dots-xuTJZ7uD7PI?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a> -->
								<!-- Photo by <a href="https://unsplash.com/@fusion_medical_animation?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Fusion Medical Animation</a> on <a href="https://unsplash.com/photos/visualization-of-the-coronavirus-causing-covid-19-rnr8D3FNUNY?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a> -->
	 							<!-- Photo by <a href="https://unsplash.com/@charlottevenema?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Charlotte Venema</a> on <a href="https://unsplash.com/photos/herd-of-sheeps-soslo4sEmBA?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a> -->
								<!-- Daniel Rosengren   -->
								<!-- Photo by <a href="https://unsplash.com/@boudewijn_huysmans?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Boudewijn Huysmans</a> on <a href="https://unsplash.com/photos/green-trees-under-white-clouds-GMjKxyV9oSo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a> -->
								<!-- Photo by <a href="https://unsplash.com/@noaa?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">NOAA</a> on <a href="https://unsplash.com/photos/underwater-photo-3nVbFN_jNNE?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a> -->
					</div>
				</div>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
